### PR TITLE
deleted two unnecessary sentences in the readme

### PR DESCRIPTION
**Pull request by qphandev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,2 @@
 # github-lesson-test
-Testing repo for demonstrations.
-
-This is a change.
-
-This is a new feature.
\ No newline at end of file
+Testing repo for demonstrations.
```
Comment (tealleaf) [inline README.md]: this code sucks, redo it

Comment (qphandev) [reply README.md]: Fixed it!

**Review by qphandev — COMMENTED** [review]
mistake

**Review by tealleaf — APPROVED** [review]
Looks good to me!

**Pull request merged**